image/mkosi/mkosi.conf.d/mkosi.conf

```diff
@@ -0,0 +1,17 @@
+[Distribution]
+Distribution=fedora
+Release=36
+
+[Output]
+Format=gpt_squashfs
```

Comment:
Is squashfs going to stay? Is compression by default enabled with gzip (as I thought it was last time)?

If so, switch to zstd if the bootloader does support that. Better compression in total and better decompression performance, which might slightly boost I/O performance from the r/o only device.

If no compression is there, carry on.